src/utils/parseError.ts

```diff
@@ -142,14 +150,22 @@ export const parseError = async (
       )
     }
 
-    if (e.Code === LifiErrorCode.TransactionUnprepared) {
+    if (e.code === LifiErrorCode.TransactionUnprepared) {
```

Comment:
Are you sure that this one here isn't supposed to be `Code`? It's not one of our errors, could be `Code`

Reply:
yes i am pretty sure
<img width="636" alt="image" src="https://user-images.githubusercontent.com/10106351/178262715-c83b19c8-8741-4320-9d13-b5e7827903cc.png">


And it is one of our errors